index.html

```diff
@@ -63,85 +65,110 @@ <h4>CANOPY</h4>
+                <li class="mobile">Back End Dev</li>
+                <li class="desktop">Full Stack Dev</li>               
                 <li>2015</li>    
             </ul>
         </div>
-            <p>
+            <p class="mobile">
                 A daily selection of privately personalized reads; no accounts or sign-ups required.
             </p>
+            <p class="desktop">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</p>
             <ul class="skills-card">
                 <li class="skill-1">html</li>
+                <li class="skill-1 desktop">Ruby on rails</li>
                 <li class="skill-1">css</li>
                 <li class="skill-1">Javascript</li>
             </ul>
-            <button class="btn"><a href="">See Project</a></button>
+            <button class="btn">See Project</button>
+    </div>
         </article>
 
-        <article class="cards">
+        <article class="cards cards-2">
             <figure>
-                <img src="images/art.png" alt="">
+                <img class="mobile" src="images/art.png" alt="">
+                <img class="desktop" src="images/desktop-version/availaibility.png" alt="">
             </figure>
-            <h2 class="title-card">Multi-Post
-                Stories</h2>    
+    <div class="detail-card">
+            <h2 class="title-card mobile">Multi-Post Stories</h2>  
+            <h2 class="title-card desktop">Uber Navigation</h2>  
         <div class="header-details">
-            <h4>CANOPY</h4>           
+            <h4 class="mobile">CANOPY</h4>
+            <h4 class="desktop">Uber</h4>          
             <ul class="dot">              
-                <li>Back End Dev</li>               
-                <li>2015</li>    
+                <li class="mobile">Back End Dev</li>
+                <li class="desktop">Lead Developer</li>
+                <li class="mobile">2015</li>    
+                <li class="desktop">2018</li>
             </ul>
         </div>
-            <p>
+            <p class="mobile">
                 A daily selection of privately personalized reads; no accounts or sign-ups required.
             </p>
+            <p class="desktop">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
             <ul class="skills-card">
                 <li class="skill-1">html</li>
+                <li class="skill-1 desktop">Ruby on rails</li>
                 <li class="skill-1">css</li>
                 <li class="skill-1">Javascript</li>
             </ul>
-            <button class="btn"><a href="">See Project</a></button>
+            <button class="btn">See Project</button>
+    </div>
         </article>
       </section>
 
       <!-- </* ----------- ABOUT MYSELF ----------- */> -->
 
       <Section class="about-myself">
```

Comment:
- [OPTIONAL ] I can see that you made a great design for this project but there are something little things you should work on for your project to look more professional, Compare your design to the Figma one, the arrows should have space between your languages' cards, kindly fix this for your project to match the Figma design. check this :point_down: 
![Screenshot from 2023-07-14 17-01-48](https://github.com/jerymey/Portfolio-setup/assets/88894525/ad3f5e11-53f0-48f0-bd48-5aceb6e12acf)
Figma Design 
 ![image](https://user-images.githubusercontent.com/69077061/253288803-08e0e2cc-7f32-43be-ab69-4fb92a52f23f.png)